portfolio/src/main/webapp/script.js

```diff
@@ -51,3 +73,27 @@ window.onclick = function(event) {
     }
   }
 }
+
+function plusSlides(n) {
```

Comment:
Separate functions for forward/back left/right, like `slideLeft()` or something, would be a little clearer and easier to maintain. In the future, someone could pass `17` or `-324` into `plusSlides()` expecting it to handle it, but it's unclear what they might be expecting it to do.